packages/snaps-sdk/src/jsx/components/Address.ts

```diff
@@ -7,9 +7,15 @@ import { createSnapComponent } from '../component';
  *
  * @property address - The (Ethereum) address to display. This should be a
  * valid Ethereum address, starting with `0x`, or a valid CAIP-10 address.
+ * @property truncate - Optional boolean flag used to control truncation of the address.
+ * @property displayName - Optional boolean flag used to control displaying a name of an account or contact.
+ * @property avatar - Optional boolean flag used to control displaying of the address avatar.
```

Comment:
WDYT ?

```suggestion
 * @property truncate - Whether to truncate the address. Defaults to `false`.
 * @property displayName - Whether to show the account name. Defaults to `false`.
 * @property avatar - Whether to show the address avatar. Defaults to `false`.
```

Reply:
I wanted to keep exactly the same functionality as it was before these changes, so I considered that `truncate` and `avatar` should be set to `true` by default. I did that in my extension PR: https://github.com/MetaMask/metamask-extension/pull/27798/files#diff-a7abb89ea6d74751032ae282629323ca2611bd2c9da06325912ffe722bf086beR34

Do you think it should be set in snaps repo? Something like `truncate: optional(defaulted(boolean(), true)),` and then `truncate?: boolean | true;` for types. 🤔 

Trying to avoid this being a breaking change for Snaps that are already using it.

Reply:
Yea, sounds good. Updated.